src/molecules/gv-row-expandable.js

```diff
@@ -71,30 +93,31 @@ export class GvRowExpandable extends LitElement {
           align-self: center;
           margin-left: 5px;
           --gv-icon--s: 16px;
-          background-color: var(--gv-theme-neutral-color-light, #efefef);
-        }
-
-        .show-more:hover {
-          background-color: var(--gv-theme-neutral-color-dark, #d9d9d9);
         }
       `,
     ];
   }
 
-  _onClick() {
+  _onClick(event) {
+    if (this.onlySummary) {
+      const summary = this.shadowRoot.querySelector('.summary');
+      if (!summary.contains(event.target)) {
+        return;
+      }
+    }
```

Comment:
It is causing troubles when set to true, a click on the `summary` slot is filtered too